src/schema/contentful/phoenix.js

```diff
@@ -657,6 +681,8 @@ const typeDefs = gql`
     ${entryFields}
   }
 
+  union ResourceWebsite = CampaignWebsite | StoryPageWebsite
```

Comment:
Let's add some documentation on this type, since it's not super clear at first glance. Maybe:

```suggestion
  "A web-based campaign interface, such as the traditional campaign template or story page."
  union ResourceWebsite = CampaignWebsite | StoryPageWebsite
```

Reply:
Dig it!